luna/featurevis/images.py

```diff
@@ -146,16 +148,6 @@ def fft_image(img, std=None, decay_power=1):
     Returns:
         New image in spatial domain.
     """
-    if tf.compat.v1.keras.backend.image_data_format() == "channels_first":
```

Comment:
I think that this check is still necessary.
Since the function is accepting a shape that can be either channel first or not (see line 116 or 118), batch, channels, height, width as a general shape becomes incorrect. 

A "channel-last" model, would yield batch, channels, height, width = 1  299 3 3 which is not correct. 
A test with 
`images.initialize_image_ref(299, 299, fft=True, decorrelate=True, seed=1)`
would produce an error.

Reply:
I had missed to push my the last commit that I think fixes this. Can you verify with the latest changes?